tox.ini

```diff
@@ -179,19 +179,19 @@ allowlist_externals =
     juju
     charmcraft
     bash
+    cp
     uv
 dependency_groups = integration
 commands =
     # Clean any stale artefacts, in case these steps are run locally and repeatedly.
-    bash -c 'rm -vf test/charms/test_smoke/*.tar.gz'
-    bash -c 'rm -vf test/charms/test_smoke/*.whl'
-    bash -c 'rm -vf test/charms/test_smoke/requirements.txt'
+    bash -c "find test/charms/test_smoke -name '*.whl' -or -name '*.tar.gz' -or -name pyproject.toml -or -name uv.lock | xargs -r rm -vf"
     # Build ops and friends from this checkout.
-    uv build --all --out-dir=test/charms/test_smoke/
+    uv build --out-dir=test/charms/test_smoke --wheel .
```

Comment:
I assume the removal of `--all` is because the smoke test only actually needs ops, not ops-tracing and certainly not ops-scenario. If so, then probably the comment above (the "and friends") should be updated too.